src/pages/Landing.tsx

```diff
@@ -31,7 +31,8 @@ const pageBox = classnames(
   display('flex'),
   flexDirection('flex-col'),
   alignItems('items-center'),
-  margin('mb-10')
+  margin('mb-10'),
+  width('w-full')
```

Comment:
Why do we need to change something on landing? This might affect small devices and some breakpoints/animations/etc

Reply:
This fixes an error with the SuperHr component, but I'll remove this, log this in trello, and do it in a separate PR instead actually so that we can test it thoroughly

The error is that without the div taking up the whole page width, SuperHr doesn't actually span the whole screen. Also it should totally have been width('w-screen') instead